llvm-remote-index.md

```diff
@@ -0,0 +1,67 @@
+# LLVM remote index service
+
+There's currently a test instance of clangd remote-index for LLVM project at
+`llvm.clangd-index-staging.dev`. The service is maintained at best effort,
+and is not an official approved by LLVM yet.
+
+After getting a clangd with remote-index support as described in
+[here](./remote-index.md#getting-remote-index-support-in-clangd), you can
+point it at `llvm.clangd-index-staging.dev:50051` either through
+[config](./config.md) or [command line
+flags](`--remote-index-address=llvm.clangd-index-staging.dev:50051` and
```

Comment:
something seems to be off here. i suppose you wanted to link to details rather than providing them here?